quickevent/app/quickevent/plugins/Oris/src/alphatoioc.h

```diff
@@ -0,0 +1,250 @@
+#ifndef ALPHATOIOC_H
+#define ALPHATOIOC_H
+
+#include <QMap>
+#include <QString>
+
+QMap<QString, QString> isoAlpha2ToIoc = {
```

Comment:
Ja nevim, fixni seznam kodů v zdrojaku. Jak budeme řešit když dojde k nějaké opravě nebo úpravě kodů?
Názvy zemí se v průběhu času vyvíjejí.

Reply:
I suppose we dont use any packaging system in this c++ project that would handle this. Git submodule for such a small thing would be overkill. The countries participating in orienteering are hardly going to change to have a meaningful impact, but I am open to suggesting how to handle this better.